scripts/tasks.sh

```diff
@@ -68,10 +69,16 @@ function tasks() {
             install_host_dependencies_longhorn $@
             ;;
         rook-10-to-14|rook_10_to_14)
-            rook_tasks_10_to_14
+            rook_upgrade_tasks_rook_upgrade "from-version=1.0" "to-version=1.4"
```

Comment:
I think you still need the wrapper here to load the util binaries

Reply:
i moved that into the function

https://github.com/replicatedhq/kURL/pull/3787/files#diff-63c30f03d53ee5e7fe2fdd09e83185a29f27ad798739c19b64bc29dcc8cfadffR463